apps/storybook/src/Table.stories.tsx

```diff
@@ -1702,21 +1702,44 @@ export const Full2: Story<Partial<TableProps>> = (args) => {
     };
   }, []);
 
+  const [globalFilterValue, setGlobalFilterValue] = useState('');
+
   return (
-    <Table
-      columns={columns}
-      emptyTableContent='No data.'
-      isRowDisabled={isRowDisabled}
-      isSelectable
-      isSortable
-      isResizable
-      enableColumnReordering
-      {...args}
-      data={data}
-      style={{ height: '100%' }}
-      enableVirtualization
-      rowProps={rowProps}
-    />
+    <div
+      style={{ display: 'flex', flexDirection: 'column', maxWidth: '1000px' }}
+    >
+      <div style={{ display: 'flex', margin: '8px 0' }}>
+        <Input
+          placeholder='Search...'
+          value={globalFilterValue}
+          onInput={(e: React.ChangeEvent<HTMLInputElement>) =>
+            setGlobalFilterValue(e.target.value)
+          }
+        />
+      </div>
+      <div
```

Comment:
it's a lot of extra divs. any chance to reduce the number?

or even just revert the story change.

Reply:
Just copy pasted from the story before..
Do we really care about divs in here..?

Reply:
this code is visible to users, so we do care a bit. but minor

Reply:
ah ok.. one div can be removed, mb